### PR TITLE
TINY-3917: Added docs for the new placeholder setting

### PR DESCRIPTION
**Pull request by lnewson**

### CODE REVIEW

File: _includes/configuration/placeholder.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## placeholder
+
+This option adds placeholder content that will be shown when the editor is empty.
+
+Alternatively, the `placeholder` attribute on the original DOM element can be used instead, if the placeholder setting isn't specified.
```
Comment (EkimChau) [inline _includes/configuration/placeholder.md]: nice!

Comment (spocke) [reply _includes/configuration/placeholder.md]: Is the commas really needed here it's not an insertion?

Comment (tiny-james) [reply _includes/configuration/placeholder.md]: I would rephrase it to:
`Alternatively, the \`placeholder\` attribute on the original DOM element will be used when the placeholder setting isn't specified.`

Comment (lnewson) [reply _includes/configuration/placeholder.md]: I did consider that, but I wanted it to be phrased so that it's something the user should add if they want this, not that it's a fallback.

Comment (tylerkelly13) [reply _includes/configuration/placeholder.md]: `If the placeholder setting isn't specified, the `placeholder` attribute on the original DOM element will be used.`
???

Comment (lnewson) [reply _includes/configuration/placeholder.md]: That's exactly what I didn't want it to say though? It should be that you can add it not, that it'll fallback.

Comment (tylerkelly13) [reply _includes/configuration/placeholder.md]: Collaborative effort from @tylerkelly13 @metricjs @lnewson 
`If the editor is initialized on a `textarea` element, the placeholder attribute can be used instead.`

**Comment by tylerkelly13** [comment]
Do we know if this will be released with TinyMCE 5.1 or 5.2?

**Comment by lnewson** [comment]
> Do we know if this will be released with TinyMCE 5.1 or 5.2? @lnewson ?

It'll be 5.2, so just updated the milestone.

**Comment by tylerkelly13** [comment]
Thank @lnewson !
